src/shared/components/FormComponents/SelectController/SelectController.tsx

```diff
@@ -165,8 +166,18 @@ export const SelectController = <T extends FieldValues>({
               'data-testid': `${dataTestid}-dropdown`,
             },
           },
-          IconComponent: (props) => <Svg className={props.className} id="navigate-down" />,
+          IconComponent: shouldSkipIcon
+            ? undefined
+            : (props) => <Svg className={props.className} id="navigate-down" />,
           ...SelectProps,
+          ...(shouldSkipIcon && {
+            inputProps: {
+              sx: {
+                pr: '1.2rem !important',
+                minWidth: '94% !important',
```

Comment:
have you tried increase specificy? like
`inputProps: {
  sx: {
    '&&': { pr: ..., minWidth: ... }
  }
}`
?

Reply:
I've checked even with '&&' specified version without '!Importtant' does not work correctly with ellipsis